test/lib/LocalizationLoader.js

```diff
@@ -453,13 +452,14 @@ describe('server/LocalizationLoader', function () {
 								'Response should have one cookie setup'
 							);
 
-							assert.strictEqual(/^locale=ru/
+							// TODO: refactoring, old value 'locale'
```

Comment:
I change `assert.strictEqual(/^locate=ru/` to `assert.strictEqual(/^testName=ru/`for correct test and I doubt it.